src/free.rs

```diff
@@ -128,17 +128,24 @@ pub fn zip<I, J>(i: I, j: J) -> Zip<I::IntoIter, J::IntoIter>
     i.into_iter().zip(j)
 }
 
-/// Create an iterator that first iterates `i` and then `j`.
+/// Returns an iterator combining two iterators consecutively. 
 ///
 /// [`IntoIterator`] enabled version of [`Iterator::chain`].
 ///
+/// ## Example
 /// ```
 /// use itertools::chain;
+/// 
+/// let a = [1, 2, 3];
+/// let b = [4];
+/// let mut result:Vec<i32> = Vec::new();
 ///
-/// for elt in chain(&[1, 2, 3], &[4]) {
-///     /* loop body */
+/// for element in chain(&a , &b) {
+///     result.push(*element);
 /// }
+/// assert_eq!(result, vec![1, 2, 3, 4]);
 /// ```
+#[deprecated(note="Use std::iter::Chain instead", since="1.0.0")]
```

Comment:
Please remove this line, as `std::iter::Chain` refers to the `struct`, but not to a function (which does (not yet) exist).

Reply:
Implemented suggestion.

Reply:
I'm not sure if your commits went in as you thought. I think we should not add the `deprecatted` notice.

Reply:
While figuring out how to properly rebase and push I ended up having to revert a few times, this line must have snuck through. :+1: